README.md

```diff
@@ -19,7 +19,7 @@ Website: https://fairlearn.github.io/
 
 ## Current release
 
-- The current stable release is available at [Fairlearn v0.5.0](https://github.com/fairlearn/fairlearn/tree/release/v0.5.0).
+- The current stable release is available at [Fairlearn v0.5.0](https://github.com/fairlearn/fairlearn/tree/release/v0.6.0).
```

Comment:
The string shown will still say v0.5.0 right now.

Reply:
Arghhh.... Thanks for catching that.